sdk/identity/identity/test/authTestUtils.ts

```diff
@@ -13,35 +13,36 @@ export interface MockAuthResponse {
 }
 
 export interface MockAuthHttpClientOptions {
-  authResponse?: MockAuthResponse,
+  authResponse?: MockAuthResponse | MockAuthResponse[],
```

Comment:
Here we're mocking the responses - it makes me wonder if it's better to keep mocking or to use record/playback and have a session to replay.  Curious what you think.

Reply:
I definitely want to add record and replay across the board for the credential tests, planning to do that in a separate PR before the end of preview.2